### PR TITLE
Response to reviews 2-3 & 2-20: dot plot & interactive HTML

### PR DESCRIPTION
**Pull request by dhimmel**
Response to reviewer 2 comment 20
Closes https://github.com/greenelab/meta-review/issues/149

Response to reviewer 2 comment 3
Interactive HTML discussion
Closes https://github.com/greenelab/meta-review/issues/132

### CODE REVIEW

**Comment by dhimmel** [comment]
Ready for review

File: content/02.main-text.md [hunk 1]
```diff
@@ -152,6 +152,7 @@ Pandoc supports conversion between additional formats — such as LaTeX, AsciiDo
 [Journal Article Tag Suite](https://jats.nlm.nih.gov/) (JATS) is a standard XML format for scholarly articles that is used by publishers, archives, and text miners [@url:http://www.niso.org/standards/z39-96-2015/; @doi:10.6087/kcse.2014.1.99; @doi:10.1080/00987913.2012.10765464].
 Pandoc's JATS support provides an avenue to integrate Manubot with the larger JATS ecosystem.
 For now, the primary Manubot output is HTML intended to be viewed in a web browser.
+Accordingly, manuscripts natively support javascript and can thus include any web-based interactive visualization.
```
Comment (agitter) [inline content/02.main-text.md]: Is JavaScript the formal capitalization?

Do we want to name any specific interactive visualization libraries here?  I'm not sure a reader will read `web-based interactive visualization` and think of graphics like what plotly, D3, Bokeh, etc. offer.

File: content/response-to-reviewers.md [hunk 2]
```diff
@@ -31,6 +31,18 @@ Since the paper has been submitted as a Software paper, maybe it's fine but it w
...
 Being able to produce both a nice PDF and a HTML dynamic version would be a nice feature.
 
+The HTML-first workflow, where the PDF is created from the HTML, does permit incorporation of interactive web visualizations, as we now note in the manuscript (edits in [GH171](https://github.com/greenelab/meta-review/pull/171)).
+One downside is that it can [be difficult](https://github.com/manubot/rootstock/issues/192 "Increasing the density of PDF exports to reduce number of pages") to create heavily typeset and whitespace-optimized PDFs using CSS.
+However, Manubot envisions a future where the HTML view is the primary means of accessing the article.
+See [GH132](https://github.com/greenelab/meta-review/issues/132) for additional discussion.
+
+Regarding interactive notebooks, some projects (e.g. [_eLife_'s reproducible article](https://elifesciences.org/labs/ad58f08d/introducing-elife-s-first-computationally-reproducible-article)) are exploring incorporating customizable source code execution directly into a manuscripts HTML view.
```
Comment (agitter) [inline content/response-to-reviewers.md]: We could add this citation to the existing Stencila citation we have.

Add apostrophe to `manuscript's`

File: content/response-to-reviewers.md [hunk 3]
```diff
@@ -31,6 +31,18 @@ Since the paper has been submitted as a Software paper, maybe it's fine but it w
...
+One challenge of this approach is that most analysis require substantial code, scattered across many layers of dependencies and inputs.
```
Comment (agitter) [inline content/response-to-reviewers.md]: `most analyses`

File: content/response-to-reviewers.md [hunk 4]
```diff
@@ -90,6 +102,10 @@ See for example https://swcarpentry.github.io/git-novice/01-basics/index.html
 > Figure .2 → This could be probably adapted as a raster plot where a dot is painted for each contribution (at the time of contribution).
 Line would be the same height and the legend would summarize the contribution.
 
+Based on this suggestion, we created an [alternative visualization](https://github.com/greenelab/meta-review/blob/6e7458dcb205a3d76daf25ee08236c24dc4de66e/analyses/deep-review-contrib/deep-review-contribution-dot.png) of Deep Review contributions that uses dots of varying sizes to show word counts by contributor on a monthly basis.
+While we thought the dot plot was a valuable supplement, there was not strong support for switching from the ridgeline plot to the dot plot for the primary manuscript visualization.
```
Comment (agitter) [inline content/response-to-reviewers.md:115]: I suggest following this sentence by one more that notes that our new Figure 2 caption alerts readers they can open the notebook in Binder to view the alternative visualizations or access the underlying data to make their own.

**Review by agitter — CHANGES_REQUESTED** [review]
These updates address the reviewer comments well, and I only have minor suggestions.

**Comment by dhimmel** [comment]
@agitter take a look again, especially wrt reorganization in https://github.com/greenelab/meta-review/pull/171/commits/0c3f05612d74210db1781694af0fbf8a8e9adde0

**Review by agitter — APPROVED** [review]

**Pull request merged**